osgi/de.cau.cs.kieler.spviz.osgi.generate/osgi.spviz

```diff
@@ -14,35 +14,66 @@ SPViz OSGiViz {
 		connect OSGi.ServiceComponent.Required via OSGi.Bundle in Services
 		connect OSGi.ServiceInterface.ProvidedBy via OSGi.Bundle in Services
 	}
-	BundleDependencies {
+	
+	RequireBundleDependencies {
+		show OSGi.Bundle
+		connect OSGi.Bundle.Dependency
+	}
+	
+	BundlePackageDependencies {
+		show OSGi.Bundle
+		show OSGi.Package
+		connect OSGi.Bundle.PackageExportDependency
+		connect OSGi.Bundle.PackageImportDependency
+		connect OSGi.Bundle.PackageDependency
+	}
+	
+	AllDependencies {
 		show OSGi.Bundle
 		connect OSGi.Bundle.Dependency
-//		connect OSGi.Bundle via OSGi.Bundle.PackageDependency
+		connect OSGi.Bundle.PackageDependency
 	}
+	
 	Products {
 		show OSGi.Product
-//		connect OSGi.Bundle.Dependency via OSGi.Product>OSGi.Feature>OSGi.Bundle
```

Comment:
This comment was also left in this file intentionally. Connecting products (or connections across multiple levels of hierarchy) can conceptually be added to SPViz, but are not implemented yet, this is one of the reminders of what could be possible with SPViz.